ui/src/app/shared/header/header.component.ts

```diff
@@ -76,8 +79,7 @@ export class HeaderComponent implements OnInit, AfterViewInit, AfterViewChecked,
       this.chips = URLSearchParamsUtils.getChips(this.route.snapshot.queryParams['q']);
     }
 
-    this.options = URLSearchParamsUtils.getQueryFields(
-      this.capabilitiesService.getCapabilitiesSynchronous());
+    this.options = URLSearchParamsUtils.getQueryFields(this.capabilities);
```

Comment:
nit: I probably prefer the previous way to avoid adding more instance variables

Reply:
makes sense